utils/src/test/scala/com/salesforce/op/utils/json/JsonUtilsTest.scala

```diff
@@ -79,20 +79,50 @@ class JsonUtilsTest extends PropSpec with PropertyChecks with TestCommon {
     }
 
   property("handle random entries correctly") {
-    forAll(dataGen)(check)
+    forAll(dataGen)(checkJson)
+    forAll(dataGen)(checkYaml)
   }
 
   property("handle special entries correctly") {
-    forAll(specialDataGen)(check)
+    forAll(specialDataGen)(checkJson)
+    forAll(specialDataGen)(checkYaml)
+
   }
 
   property("handle empty collections correctly") {
-    forAll(doubles) { d => check(TestDouble(d, Array.empty, Seq.empty, Map.empty, None)) }
+    forAll(doubles) { d => checkJson(TestDouble(d, Array.empty, Seq.empty, Map.empty, None)) }
+    forAll(doubles) { d => checkYaml(TestDouble(d, Array.empty, Seq.empty, Map.empty, None)) }
+  }
+
+  property("read json file") {
+    val jsonFile = resourceFile(name = "Person.json")
+    val person = Person("Foo Bar", Address("Talkeetna", "AK"))
+    JsonUtils.fromFile[Person](jsonFile) match {
+      case Failure(e) => fail(e)
+      case Success(value) => assert(value, person)
+    }
+  }
+
+  property("read yml file") {
+    val ymlFile = resourceFile(name = "Person.yml")
+    val person = Person("Foo Bar", Address("Talkeetna", "AK"))
+    JsonUtils.fromFile[Person](ymlFile) match {
+      case Failure(e) => fail(e)
+      case Success(value) => assert(value, person)
```

Comment:
since it's a case class you dont need a special assertion method but you can use the equality directly, i.e `value shouldBe person` - http://www.alessandrolacava.com/blog/scala-case-classes-in-depth/

Reply:
Good to know that.Thanks. Updated!